agent/util-scripts/pbench-tool-data-sink

```diff
@@ -22,6 +22,10 @@ import subprocess
 import sys
 import tempfile
 
+#import importlib
+#toolmeister = importlib.import_module("pbench-tool-meister")
+from modules import metaclass
```

Comment:
Did you mean to use this import?

Reply:
I wonder what `flake8` reports for this.